### ISSUE TITLE
IFC4 MEP certification

### ISSUE THREAD

**Issue by HL-code** [Autodesk/revit-ifc#357]
Hi

Do you have a status or timeline on the Revit IFC4 mep certification. (Export)
When can we expect the Revit export to be certified for  IFC4 MEP ?

Cheers
Håvard Leding
Symetri

**Comment by AngelVelezSosa** [comment]
I can't give you a date for certification, but I can tell you our current progress.
Out of 16 tests, we have created 14 test models and are starting work on the last 2.  Of the 14 created, 11 have passed automated testing, 2 have a small number of issues we are investigating (unclear if they are IFC issues, buildingSMART issues, or model creation issues) and 1 we just finished so need to run through the automated testing.  When everything passes automated testing, we will run it through the automated testing again to make sure there are no regressions (hopefully, that will take minutes, or days or weeks), and then submit one or two files for manual testing.  Assuming that goes well, we will send the rest of manual testing.

We will likely do an updated for when we (1) pass automated testing and (2) pass manual testing.
